content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx

```diff
@@ -19,17 +19,47 @@ We are continuously testing the Data Proxy with Cloudflare Workers. [Open an iss
 
 </TopBlock>
 
-
 ## Prerequisites
 
-- Free [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) database with an accessible URL
-  - e.g. `mongodb+srv://<user>:<password>@cluster.mongodb.net`.
-- Free [Cloudflare Worker](https://workers.cloudflare.com/) account.
-- Free [GitHub](https://github.com/) account.
-- Node.js & NPM installed.
-- Git installed.
+- Free [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) account
+- Free [Cloudflare Workers](https://workers.cloudflare.com/) account
+- Free [GitHub](https://github.com/) account
+- Node.js & NPM installed
+- Git installed
+
+## 1. Get a MongoDB Atlas connection string
```

Comment:
Are we sure there is no better way to get a connection string with a database name in Atlas?

Reply:
When you've created a cluster in Atlas, you can click **Connect**.

This opens a dialog window with 4 options:

<img width="808" alt="00-mongodb-conn-string-options" src="https://user-images.githubusercontent.com/103522279/187205108-bc541c24-2032-4dcf-b44c-39a996ac4963.png">

* **Connect with the MongoDB Shell**
This shows steps with MongoDB Shell and suggests this string:
`mongosh "mongodb+srv://cluster0.3ptpdyy.mongodb.net/myFirstDatabase" --apiVersion 1 --username <username>`
The suggested database name is `myFirstDatabase`.
<img width="808" alt="01-mongodb-shell-conn-string" src="https://user-images.githubusercontent.com/103522279/187204120-92eb79b0-65d2-4e00-bcea-dcc4818a904a.png">

This one has the clearest explanation about where the database name goes, but I never considered it because I wasn't using MongoDB Shell.
* **Connect your application**
You can select a **Driver** and **Version**, such as **Node.js** and **4.1 or later**. That's what I click every time because based on my use case it seems to be the most relevant.
The suggested connection string is this:
`mongodb+srv://<username>:<password>@cluster0.3ptpdyy.mongodb.net/?retryWrites=true&w=majority`

<img width="808" alt="02-application-conn-string" src="https://user-images.githubusercontent.com/103522279/187204241-d32c2f4b-9427-4b54-b3b2-280db963ff64.png">

This has been confusing me the most and it's what I have used every time.

* **Connect using MongoDB Compass**
This asks for the installation of Compass, and the suggested connection string is 
`mongodb+srv://zti:<password>@cluster0.3ptpdyy.mongodb.net/test`
To me, it is not obvious from here what `test` is.

<img width="809" alt="03-mongodb-compass-conn-string" src="https://user-images.githubusercontent.com/103522279/187204930-e730b63b-cfe0-4e8f-b260-ff5713c1b1db.png">


* **Connect using VS Code**
This one does include a database name in the conn. string but it fails to explain that from the UI.

<img width="808" alt="04-vscode-conn-string" src="https://user-images.githubusercontent.com/103522279/187204697-ce93dcdd-f199-46bd-80ad-d354f9f66e19.png">